### ISSUE TITLE
Warning message about integer overflow when loading tidyverse in a fresh R session

### ISSUE THREAD

**Issue by thanosgatos** [tidyverse/tidyverse#150]
OS: Ubuntu 18.04.1 LTS
R version: 3.5.1
tidyverse version: 1.2.1
All packages are up-to-date.

For the last couple of days, I have noticed the following warning message pop up the first time I load tidyverse, as well as the first time I'm calling some other functions--if my memory is not failing me, this was happening yesterday with `read_csv()`.

The warning message I get is the following:
```
Warning message:
In n * h : NAs produced by integer overflow
```

The weirdest part is that, after I restart R, using Ctrl+Shift+F10, I do not get the same warning when loading tidyverse.

**Comment by batpigandme** [comment]
Hmm— NAs produced by integer overflow usually refers to the attempt to coerce a value to an integer. Are you opening some sort of saved workspace? 
Have you recently updated the tidyverse package?
What happens if you load another package first? 
It makes more sense with `read_csv()` as, in so doing you're reading in values…

See https://github.com/tidyverse/purrr/issues/403 for an example, and/or this thread, [What is integer overflow in R and how can it happen?](https://stackoverflow.com/questions/8804779/what-is-integer-overflow-in-r-and-how-can-it-happen), on StackOverflow.

Since tidyverse is a meta-package, it seems unlikely that whatever is producing this is internal to the specific package. You might want to ask about this on [community.rstudio.com](https://community.rstudio.com/) (along with some of the answers to the questions I listed above) where more people can help you troubleshoot (and if it _does_ turn out to be a tidyverse package bug, we can address it with more information here).

**Comment by thanosgatos** [comment]
Thanks for the quick reply.

Answering your questions:
1. I am never saving my workspace.
2. I do not remember when was the last time I've updated tidyverse.
3. Every single package I've tried, after opening RStudio, gives me the same warning. Therefore, I tend to think it is something related to the IDE--I've been running the 1.2.830 preview, but I've just seen there is a newer one. 

Thank you for the provided links. I had already been reading through them, but my case seemed very weird. 

Given the 3rd point above, feel free to close this issue. I will have yet another look on RStudio Community and, if the update to the latest RStudio preview does not fix this, open an issue in rstudio/rstudio.
